src/components/organisms/AdminDashboard/AdminDashboard.tsx

```diff
@@ -68,7 +69,7 @@ export const AdminDashboard: React.FC = () => {
             <div className="AdminDashboard__header-title">Admin Dashboard</div>
             {sortingOptions}
           </div>
-          <ButtonNG variant="primary" isLink linkTo={ADMIN_CREATE_SPACE_URL}>
+          <ButtonNG variant="primary" isLink linkTo={ADMIN_V3_CREATE_URL}>
```

Comment:
OOS `isLink` is redundant and `linkTo` can be used inside the component as a conditional check or whatever it needs to do.

Reply:
The idea for `ButtonNG` is simple:

`isLink` - should I use `<a>` or should I use `<button>`
`linkTo` - no matter if `<a>` or `<button>`, upon click, navigate to the provided URL

Maybe the time has come to expand on it and add the missing use case.

This has been also a need for few BM use-cases (slack reference for one of those https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1630255107051100?thread_ts=1630253800.048900&cid=C01ER4M61TL ).

Reply:
Then we should create another component called `Link` or `Anchor`. It is very miss-leading. Right now we have a component called `ButtonNG` but it's not always rendering a button. 🧠 💣 I think we should improve this a bit. Still OOS tho.

Reply:
Here is a PR for that https://github.com/sparkletown/sparkle/pull/2326